src/api/app/helpers/webui/user_helper.rb

```diff
@@ -1,4 +1,24 @@
 module Webui::UserHelper
+  def display_filters?(filters)
+    return false if filters.empty?
+    return false if filters.size == 1 && filters.include?('search_text')
+
+    true
+  end
+
+  def filters_to_humans(filters)
```

Comment:
:thought_balloon: I guess you want to say `filters_to_human` (singular) like [rails does](https://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html#method-i-number_to_human)?

Reply:
actually I'm accepting suggestions for this method name. Any better idea @danidoni ?

Reply:
Now its better